website/content/docs/release-notes/consul-api-gateway/v0_4_x.mdx

```diff
@@ -0,0 +1,72 @@
+---
+layout: docs
+page_title: 0.4.x
+description: >-
+  Consul API Gateway release notes for version 0.4.x
+---
+
+# Consul API Gateway 0.4.0
+
+## Release Highlights
+
+- **Support for Kubernetes Gateway API Version 0.5.0 and v1beta1 APIs:**
+  The `v0.5.0` release of the Kubernetes Gateway API is significant because it
+  marks the growth in maturity to a beta API version (v1beta1) release for some
+  of the key APIs:
+  - GatewayClass
+  - Gateway
+  - HTTPRoute
+
+  The other APIs (e.g. TCPRoute) are still at the `v1alpha2` stage.
+
+  Reaching `v1beta1` status has several benefits for users, including greater
+  stability and backward compatibility requirements. Existing fields and allowed
+  options can not be removed or renamed except in a new, major version of the
+  API. Once an API reaches `v1beta1` status, future versions must comply with
+  several backward compatibility requirements.
+
+- **URL Path Prefix Rewrite**
+  This release introduces support for rewriting a URL's path prefix when routing
+  HTTP traffic. This is configured by adding a `URLRewrite` filter to a
+  `HTTPRoute`. With this feature, the gateway can rewrite the URL path, in a
+  client's HTTP Request, before sending the request to a service. A simple
+  example of this is changing the path from `//store/checkout` to
+  `//cart/checkout`. Please see the product documentation for details on how to
+  configure this feature.
+
+## What's Changed
```

Comment:
```suggestion
## Changes to the Kubernetes API specification
```
We should avoid contractions; especially in headings. Is this an accurate heading? OR should we just call it "Changes"?

Reply:
```suggestion
## What Has Changed
```
This heading is actually in the Release Notes template and should stay consistent across Rel Notes (i.e. not be K8s specific).